### DISCUSSION TITLE
suggestion for config file under Windows platform

### DISCUSSION THREAD

**Discussion by SoftwareMagicIT** [rustdesk/rustdesk#1278] [General]
Although I looked at the code I didn't understand how to do it, but I suggest moving the configuration file from appdata \ roaming \ rustdesk to programdata \ rustdesk so that it is not tied to a specific user.

**Comment by feltel** [comment]
I second this idea. We are migrating away from AnyDesk and want to self-host Rustdesk. The Windows client places its config files under %AppData% per user. We have to control certain settings machine wide (approve-mode, IP whitelist, ...) as doing it on an per-user base is not an option. Some machines are in the field not joined to an AD and setting those options via our software deployment tool is doable (ie. copying a config file to %ProgramData%\...) but we dont always have control on which user logs on to which machine. So we cannot control individual %AppData% config files.

Installing and using it as a service (which has an central config file) is not an option. Rustdesk should be started and used only when the individual user has an support case and wants an remote session.
